cmd/operator/deploy/operator/12-alertmanager.yaml

```diff
@@ -65,9 +65,18 @@ spec:
         args:
         - --config.file=/alertmanager/config_out/config.yaml
         - --storage.path=/alertmanager-data
+        - --cluster.listen-address=[$(POD_IP)]:9094
+        - --web.listen-address=:9093
+        - --web.route-prefix=/
```

Comment:
Is this line and the line before necessary for this change?

Reply:
Probably not `--web.route-prefix`, but I'd prefer to be explicit about the `--web.listen-address` as the config-reloader container [hardcodes](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/pintohutch/am-fix/cmd/operator/deploy/operator/12-alertmanager.yaml#L102-L103) that port in its flags.

Removed the last line.